src/main/resources/messages.properties

```diff
@@ -1,5 +1,27 @@
 welcome = Welcome to main page
-
+homepage = Homepage
```

Comment:
i18n is not working on login/registration page

Reply:
Login and registration pages in another tasks